frontend/src/components/BookInfo.js

```diff
@@ -1,33 +1,33 @@
 import React, { Component } from 'react';
+import { URLParamToString } from '../utils/formatHelpers';
 import '../styles/BookInfo.scss';
+
 class BookInfo extends Component {
   render() {
     return (
       <div className="blue-gradient book-info">
         <div className="container">
           <div className="row book-info-row  font-white">
             <div className="col col-lg-3">
-              <img
-                alt="cannot be displayed"
-                className="book-cover img-fluid"
-                src={this.props.bookObject.cover_url}
-              />
+              {undefined && (
+                <img
+                  alt="cannot be displayed"
+                  className="book-cover img-fluid"
+                  src={this.props.bookObject.cover_url}
+                />
+              )}
```

Comment:
not entirely sure what this does, but wouldn't this never display the image?

Reply:
orip yeah this needs to be changed